sdks/python/apache_beam/testing/benchmarks/cloudml/cloudml_benchmark_test.py

```diff
@@ -32,6 +35,27 @@
 _OUTPUT_GCS_BUCKET_ROOT = 'gs://temp-storage-for-end-to-end-tests/tft/'
 
 
+def _publish_metrics(pipeline, metric_value, metrics_table):
+  influx_options = InfluxDBMetricsPublisherOptions(
+      metrics_table,
+      pipeline.get_option('influx_db_name'),
+      pipeline.get_option('influx_hostname'),
+      os.getenv('INFLUXDB_USER'),
+      os.getenv('INFLUXDB_USER_PASSWORD'),
+  )
+  metric_reader = MetricsReader(
+      project_name=pipeline.get_option('project'),
+      bq_table=metrics_table,
+      bq_dataset=pipeline.get_option('metrics_dataset'),
+      publish_to_bq=True,
+      influxdb_options=influx_options,
+  )
+  metric_reader.publish_values([(
+      'runtime',
+      metric_value,
```

Comment:
1) 
it would help to include the units, such as: runtime_sec in metric name.
2) do we publish to influx db just to have grafana dashboards?
3) should this helper be defined externally to the test? Is it something we can reuse across multiple benchmark files (parameterized by some test id?)
4) looks like in only publishes runtime, let's call it `publish_runtime()`.

Reply:
1. Changed it.
2. Yes, that is right, once the changes are in, I will create a grafana dashboard.
3. I am not sure if that would be useful. MetricsReader is all we need and to publish it to influx DB we would need InfluxDB related information, which I passed it through pipeline options. Other benchmark tests, they have an abstract class `LoadTest`, which takes care of these things. Other benchmark test like fastavro_it_test do it like the way I did it.
4. Added a param to the function to accept any metric name.